types/rdf-ext/index.d.ts

```diff
@@ -1,31 +1,28 @@
 import { Environment } from "@rdfjs/environment/Environment.js";
-import { FormatsFactory } from "@rdfjs/environment/FormatsFactory.js";
 import { FetchFactory } from "@rdfjs/fetch-lite/Factory.js";
+import { FormatsFactory } from "@rdfjs/formats/Factory.js";
 import { NamespaceFactory } from "@rdfjs/namespace/Factory.js";
 import { PrefixMapFactory } from "@rdfjs/prefix-map/Factory.js";
 import { ScoreFactory } from "@rdfjs/score/Factory.js";
 import { TermMapFactory } from "@rdfjs/term-map/Factory.js";
 import { TermSetFactory } from "@rdfjs/term-set/Factory.js";
 import { TraverserFactory } from "@rdfjs/traverser/Factory.js";
-import ClownfaceFactory from "./ClownfaceFactory.js";
 import { DataFactoryExt as DataFactory } from "./DataFactory.js";
 import { DatasetFactory } from "./DatasetFactory.js";
 
 export { default as Environment } from "@rdfjs/environment/Environment.js";
-export { default as FormatsFactory } from "@rdfjs/environment/FormatsFactory.js";
 export { default as FetchFactory } from "@rdfjs/fetch-lite/Factory.js";
```

Comment:
Dont you need to use newly added d.ts file instead ?

Reply:
You mean `FormatsFactory`? I export it on line 23